curr_scraper/settings.py

```diff
@@ -11,8 +11,10 @@
 """
 
 from pathlib import Path
+from dotenv import load_dotenv
 import os
 import sys
```

Comment:
Follow the import style described in PEP8 documentation: https://peps.python.org/pep-0008/#imports

First of all split imports into following groups devided by additional newline: builtin, django imports, third-party, local imports
Additionally, sort general imports (imports without "from") first, then imports with from.
Moreover, sort alphabetically

Reply:
I believe this is fixed now.

Reply:
Good job!